### DISCUSSION TITLE
XiaomiKey BL 锁秒解价格统计

### DISCUSSION THREAD

**Discussion by MiFanLover1** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#186] [General]
### 喜报
内鬼开设的 XiaomiKey BL 秒解服务目前最新价格 120 刀！！
<div align="center"><img src="https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/159231168/d664f710-bc43-4c84-b8a0-9eebfe711e22" width="40%"></div>
<br />
<table align="center">
    <tr>
        <th>日期</th>
        <th>美元价格</th>
        <th>换算人民币价格</th>
    </tr>
    <tr>
        <td>2024-06-03</td>
        <td>USD $120.00</td>
        <th>RMB ¥869.45</th>
    </tr>
    <tr>
        <td>2024-04-27</td>
        <td>USD $100.00</td>
        <th>RMB ¥724.54</th>
    </tr>
    <tr>
        <td>2024-04-08</td>
        <td>USD $95.00</td>
        <th>RMB ¥688.31</th>
    </tr>
    <tr>
        <td>2024-03-21</td>
        <td>USD $75.00</td>
        <th>RMB ¥543.41</th>
    </tr>
    <tr>
        <td>2024-02-18</td>
        <td>USD $65.00</td>
        <th>RMB ¥470.95</th>
    </tr>
    <tr>
        <td>2024-02-13</td>
        <td>USD $50.00</td>
        <th>RMB ¥362.27</th>
    </tr>
    <tr>
        <td>2024-01-27</td>
        <td>USD $61.99</td>
        <th>RMB ¥449.14</th>
    </tr>
    <tr>
        <td>2024-01-26</td>
        <td>USD $55.00</td>
        <th>RMB ¥398.50</th>
    </tr>
    <tr>
        <td>2024-01-23</td>
        <td>USD $38.50</td>
        <th>RMB ¥278.95</th>
    </tr>
    <tr>
        <td>2023-11-30</td>
        <td>USD $27.99</td>
        <th>RMB ¥202.80</th>
    </tr>
</table>

**Comment by kingsznhone** [comment]
建立二级市场，增加看涨期权，梭哈多空合约。
别早八答题了，直接8点开盘进场炒Key

**Reply by motanzg** [reply]
米米牛逼

**Comment by LoyaltyMu** [comment]
没想到这么贵了

**Comment by 501658362** [comment]
妈呀 之前三四百没舍得，现在都八百多了。

**Comment by Furina666** [comment]
4m了

**Comment by Furina666** [comment]
幽默内鬼，闲鱼卖解锁资格最高才500多
